tests/unit/data/_async/test_client.py

```diff
@@ -2005,21 +2005,25 @@ async def test_read_rows_sharded_negative_batch_timeout(self):
         They should raise DeadlineExceeded errors
         """
         from google.cloud.bigtable.data.exceptions import ShardedReadRowsExceptionGroup
+        from google.cloud.bigtable.data._helpers import _CONCURRENCY_LIMIT
         from google.api_core.exceptions import DeadlineExceeded
 
         async def mock_call(*args, **kwargs):
-            await CrossSync.sleep(0.05)
+            await CrossSync.sleep(0.06)
             return [mock.Mock()]
 
         async with self._make_client() as client:
             async with client.get_table("instance", "table") as table:
                 with mock.patch.object(table, "read_rows") as read_rows:
                     read_rows.side_effect = mock_call
-                    queries = [ReadRowsQuery() for _ in range(15)]
+                    num_calls = 15
+                    queries = [ReadRowsQuery() for _ in range(num_calls)]
                     with pytest.raises(ShardedReadRowsExceptionGroup) as exc:
-                        await table.read_rows_sharded(queries, operation_timeout=0.01)
+                        await table.read_rows_sharded(queries, operation_timeout=0.05)
```

Comment:
Does these updates mean async client got slower?

Reply:
The value is shared between the sync and async files, and since sync is a bit slower, I reduced the value so both tests would pass

But FWIW, this test isn't meant to test performance, it's just testing the behaviour of operations that timeout. mid-stream. So using a less aggressive value here makes sense anyway IMO